test/flask_app.py

```diff
@@ -0,0 +1,129 @@
+# Note that this file contains some inline comments starting with # --, used to
```

Comment:
Note: This file is identical to the one in examples/flask/app.py but I added a new route to test a more complex scenario(a GET endpoint that accepted a list and an object in it's query string parameters) that didn't need to be in the docs.